src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -10,24 +10,29 @@ object Serializer {
   val NewLine = '\n'
   val Indent = "  "
 
-  /** Converts a `FirrtlNode` into its string representation. */
+  /** Converts a `FirrtlNode` into its string representation
+   * with *zero* indents (old behavior for binary compatibility) */
```

Comment:
```suggestion
  /** Converts a `FirrtlNode` into its string representation with default indentation
    */
```
A couple of things here:
1. Use ScalaDoc style (stars align with the rightmost star from the start of the comment
2. Slight tweak to phrasing, the behavior here didn't change so no reason to bring attention to it. There also is indentation if you pass something with things scoped inside it (like a Module or a When), so slightly more precise phrasing.